conf/app.routes

```diff
@@ -99,11 +99,15 @@ POST       /return-change-registration                          controllers.Retu
 
 GET        /remove-small-producer-confirm/:sdil                 controllers.RemoveSmallProducerConfirmController.onPageLoad(mode: Mode = NormalMode, sdil: String)
 POST       /remove-small-producer-confirm/:sdil                 controllers.RemoveSmallProducerConfirmController.onSubmit(mode: Mode = NormalMode, sdil: String)
+GET        /remove-small-producer-confirm/:sdil                 controllers.RemoveSmallProducerConfirmController.onPageLoad(mode: Mode = CheckMode, sdil: String)
+POST       /remove-small-producer-confirm/:sdil                 controllers.RemoveSmallProducerConfirmController.onSubmit(mode: Mode = CheckMode, sdil: String)
 
 GET        /add-small-producer                                  controllers.AddASmallProducerController.onPageLoad(mode: Mode = NormalMode)
 POST       /add-small-producer                                  controllers.AddASmallProducerController.onSubmit(mode: Mode = NormalMode)
 GET        /add-small-producer-edit                             controllers.AddASmallProducerController.onEditPageLoad(mode: Mode = EditMode, sdilReference: String)
 POST       /add-small-producer-edit                             controllers.AddASmallProducerController.onEditPageSubmit(mode: Mode = EditMode, sdilReference: String)
+GET        /add-small-producer-edit                             controllers.AddASmallProducerController.onEditPageLoad(mode: Mode = CheckMode, sdilReference: String)
```

Comment:
Are these duplicates of the one above - do we need a slightly different route name?